src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,21 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <li className={classNames(
+    'TodoInfo', { 'TodoInfo--completed': todo.completed },
+  )}
+  >
+    <h2 className="TodoInfo__title">
+      {todo.title}
+    </h2>
+
+    {todo.user
+      && <UserInfo user={todo.user} />}
```

Comment:
Looks better now. There's even better way though)
```suggestion
    {todo.user && (
       <UserInfo user={todo.user} />
    )}
```